GAMMA/RTC/README_RTC_GAMMA.txt

```diff
@@ -1,11 +1,16 @@
 ASF RTC Data Package (GAMMA)
 ============================
 
-This folder contains radiometric terrain corrected (RTC) products and their associated files. They were generated by the Alaska Satellite Facility (ASF) HYP3 RTC software version [HYP3_VER], running GAMMA release [GAMMA_VER]. They are projected to [PCS], and the pixel spacing is [SPACING] m.
+<!-- Consider opening this document in a Markdown editor/viewer for easier reading -->
+
+The side-looking geometry of SAR imagery leads to geometric and radiometric distortions, causing foreshortening, layover, shadowing, and radiometric variations due to terrain slope. Radiometric terrain correction converts unprocessed SAR data into geocoded TIFF images with values directly relating to physical properties, alleviating the  inherent SAR distortions. The process improves backscatter estimates and provides geolocation information, so images can be used as input for applications such as the monitoring of deforestation, land-cover classification, and delineation of wet snow-covered areas.
```

Comment:
I would prefer that this section be further down. The idea was to start out with a very basic statement of what this product is, then get further into the weeds the further down you go in the document. I see the merit in defining RTC before its first reference, but it's a lot of words to get through before you get to the basic description of the product folder.

One suggestion is to include it in the "Using this data" section. While it doesn't fit the citations and acknowledgements aspect of that section, the user guide and ATBD are included there, so it seems like it's not too much of a stretch to add the basic overview of what RTC does. I would advocate for having it be the first thing in the Using this data section, but I'm open to other options.

**Another option, which I think might be better (and might be more palatable to Joe), is to include it in the Product Contents section. It could be first thing, and the current first line actually segues pretty nicely from the general RTC description to the contents that we include.**

Reply:
I moved it because it's a general discussion of "Why RTC" that didn't fit inside the product details (was between file naming convention and the product description, with the user guide link and the ESA citation) Likewise, I do not think if fits in the "Using this data section". 

I agree it actually could fit well in the Product Contents section, especially if we renamed it "Product Description" or "Product Details". 

Another option would be to put it at the top of the [RTC Processing](https://gist.github.com/jhkennedy/36f0fd179e2f19fc7ae6e1fc00264aed#rtc-processing) section